hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/config/OpenAPIConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.config
+
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.info.Contact
+import io.swagger.v3.oas.models.info.Info
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.info.BuildProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class OpenAPIConfiguration {
+  @Autowired(required = false)
+  private val buildProperties: BuildProperties? = null
+
+  @Bean
+  fun api(): OpenAPI {
+    return OpenAPI()
+      .info(
+        Info().title("Prisoner search API Documentation")
+          .description("API for searching for offenders in Delius.")
```

Comment:
would have been better to copy this file from indexer instead - don't like the nullable `BuildProperties` etc

Reply:
Will address the comments in a separate PR